### ISSUE TITLE
does this work with 3.7?

### ISSUE THREAD

**Issue by khblaw** [WestonSF/ArcGISFlexViewerTOCGroupWidget#2]
I downloaded the widget and stored it with my project, and added the widget info to the config file, but I get error #32. Also through the web app builder, i can see the widget, but when I try to edit it says, missing widget module, could not find TOCGroup module for layers widget. What am I missing? Will this work with 3.7?

**Comment by WestonSF** [comment]
Yeah it hasn't been setup to work with the Flex web app builder. The last version I tested it working with is 3.4 - http://mapping.gw.govt.nz/Wairarapa

I haven't really been looking at it anymore as I'm focusing more on the Web AppBuilder for ArcGIS and JavaScript API now.
